src/main/java/inu/parser/Parser.java

```diff
@@ -0,0 +1,216 @@
+
+        try {
+
+            ExceptionManager.checkEmptyUserInput(entry);
+            Todo todoTask = new Todo(entry);
+            taskList.addTask(todoTask);
+            Ui.printAcknowledgment(taskList);
+
+        } catch (EmptyUserInputException e) {
+
+            Ui.printPromptValidTaskEntry();
+
+        }
+
+    }
+
+    public static void runDeadLine(TaskList taskList, String entry) {
+
+        try {
+
+            String task = Util.fetchTask(entry);
+            String deadLine = Util.fetchDeadLine(entry);
+            ExceptionManager.checkEmptyUserInput(task, deadLine);
+            DeadLine deadLineTask = new DeadLine(task, deadLine);
+            taskList.addTask(deadLineTask);
+            Ui.printAcknowledgment(taskList);
+
+        } catch (EmptyUserInputException e) {
+
+            Ui.printPromptValidByDateEntryAfterSlash();
+
+        } catch (StringIndexOutOfBoundsException e) {
+
+            Ui.printPromptValidDeadLine();
+
+        }
+
+    }
+
+    public static void runEvent(TaskList taskList, String entry) {
+
+        try {
+
+            String task = Util.fetchTask(entry);
+            String from = Util.fetchFrom(entry);
+            String to = Util.fetchTo(entry);
+            ExceptionManager.checkEmptyUserInput(task, from, to);
+            Event eventTask = new Event(task, from, to);
+            taskList.addTask(eventTask);
+            Ui.printAcknowledgment(taskList);
+
+        } catch (EmptyUserInputException e) {
+
+            Ui.printPromptValidFromAndToEntryAfterSlash();
+
+        } catch (StringIndexOutOfBoundsException e) {
+
+            Ui.printPromptValidEvent();
+
+        }
+
+    }
+
+    public static void runDelete(TaskList taskList, String entry) {
+
+        try {
+
+            ExceptionManager.checkEmptyUserInput(entry);
+            int deleteTaskNumber = Util.fetchIndexFromString(entry);
+            Ui.printDelete(taskList, deleteTaskNumber);
+            taskList.deleteTask(deleteTaskNumber);
+
+        } catch (IndexOutOfBoundsException e) {
+
+            Ui.printPromptValidTaskIndex();
+
+        } catch (EmptyUserInputException e) {
+
+            Ui.printPromptValidDeleteEntry();
+
+        }
+
+    }
+
+    public static void runList(TaskList taskList) {
+
+        try {
+
+            ExceptionManager.checkEmptyTaskList(taskList);
+            Ui.printTaskList(taskList);
+
+        } catch (EmptyTaskListException e) {
+
+            Ui.printPromptEmptyTaskList();
+
+        }
+
+    }
+
+    public static void runMark(TaskList taskList, String entry) {
+
+        try {
+
+            ExceptionManager.checkEmptyUserInput(entry);
+            int markIndex = Util.fetchIndexFromString(entry);
+            Util.markTask(taskList, markIndex);
+            Ui.printMark(taskList, markIndex);
+
+        } catch (IndexOutOfBoundsException e) {
+            Ui.printPromptValidTaskIndex();
+        } catch (EmptyUserInputException e) {
```

Comment:
See line 130. Be consistent throughout your codebase as to whether there is an empty line on the first line of the catch block. That being said, ensure that the use of them is consistent and not excessive.